multicast/__main__.py

```diff
@@ -432,12 +426,29 @@ def setupArgs(cls, parser):
 			)
 
 	@staticmethod
-	def _help_deamon_dispatch(*args, **kwargs):
-		_useHear = kwargs.get("is_deamon", False)
+	def _help_daemon_dispatch(*args, **kwargs):
+		_useHear = kwargs.get("is_daemon", False)
 		return _useHear
 
 	def doStep(self, *args, **kwargs):
-		if self._help_deamon_dispatch(*args, **kwargs):
+		"""
+		Executes a multicast step based on the daemon dispatch.
+
+		This method selects either the `McastHEAR` or `McastRECV` class based on the daemon
+		dispatch flag and executes the corresponding step.
+
+		The RECV (via McastRECV) is the primitive sub-command to receive a single multicas hunk.
+		The HEAR (via McastHEAR) is equivalent to running RECV in a loop to continually receive
+		multiple hunks. Most use-case will probably want to use HEAR instead of RECV.
```

Comment:
```suggestion
		The RECV (via McastRECV) is the primitive sub-command to receive a single multicast hunk.
		The HEAR (via McastHEAR) is equivalent to running RECV in a loop to continually receive
		multiple hunks. Most use-case will probably want to use HEAR instead of RECV.
```